development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -51,7 +51,7 @@ Distro-specific one-liners
 |                  |                                                                                                           |
 |                  |     sudo dnf install scons pkgconfig libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel \    |
 |                  |         libXi-devel mesa-libGL-devel mesa-libGLU-devel alsa-lib-devel pulseaudio-libs-devel \             |
-|                  |         libudev-devel yasm                                                                                |
+|                  |         libudev-devel yasm gcc-c++                                                                               |
```

Comment:
That breaks the table as the trailing `|` is no longer aligned.

Reply:
Sorry about that. Fixed now.